components/Common/Badge/index.tsx

```diff
@@ -0,0 +1,55 @@
+import ArrowRightIcon from '@heroicons/react/24/solid/ArrowRightIcon';
+import Link from 'next/link';
```

Comment:
Please read the Collaborator Guidelines. We don't use next/link here, only LocalizedLink

Reply:
Thank you, I will read it again